client/PublicRequestTypes.ts

```diff
@@ -288,6 +288,13 @@ export interface EuroHtlcCreationInstructions {
     bankLabel?: string;
 }
 
+export interface SinpeMovilHtlcCreationInstructions {
+    type: 'CRC';
+    value: number; // CRC cents
+    fee: number; // CRC cents
+    recipientLabel?: string;
```

Comment:
`recipientLabel` is not good for the sender side, I suggest to use `senderLabel` or `sinpeLabel` like in the Keyguard.